src/ripple/app/tx/apply.h

```diff
@@ -39,11 +41,24 @@ class Application;
 
     @return The TER code (a `tem` or tesSUCCESS)
 */
-TER
+PreflightResult
```

Comment:
This doesn't make sense - a public API function returning a private struct? (`PreflightResult` is declared in `impl/Transactor.h`, which is a private header).